docs/config.json

```diff
@@ -424,7 +424,7 @@
               "forScopes": ["enterprise", "cloud"]
             },
             {
-              "title": "Role Requests in OSS Teleport",
```

Comment:
I think the role requests part was still valuable, as this does not apply to resource requests.